taskcluster/docker/base/Dockerfile

```diff
@@ -27,6 +27,13 @@ RUN apt-get update -qq \
                           python3-yaml \
                           locales \
                           git \
+                          tmux \
+                          htop \
```

Comment:
interesting that it requires htop

Reply:
It's not the training process itself that requires htop; that's just a useful tool when folks are using interactive tasks to debug.